### ISSUE TITLE
How about epochs training coco dataset?

### ISSUE THREAD

**Issue by ChenYingpeng** [zylo117/Yet-Another-EfficientDet-Pytorch#164]
@zylo117 Great work!
     First, how about epochs training coco dataset, default 500 epochs?
     Second, from your repo, I know that you don‘t load pretrained weights from [EfficientNet-PyTorch](https://github.com/lukemelas/EfficientNet-PyTorch),because you rewrited Conv2dStaticSamePadding and MaxPool2dStaticSamePadding by yourself. I want load pretrained weights to speed up my training,because too slow training 500 epochs(From your repo default 500 epochs).Have you test this and how about impact of mAP?Thanks in advance.

**Comment by zylo117** [comment]
training on coco takes time and requires you to change parameters frequently. The default parameters are only for the early stage. Epoch500 is just a limit, doesn't mean it need to be trained for 500 epochs

**Comment by ChenYingpeng** [comment]
How do I make sure that my training fit  and by loss value?

**Comment by zylo117** [comment]
by testing, evaluation, monitoring loss graph

**Comment by ChenYingpeng** [comment]
Could you share me your training log when training EfficientDet-d0 in coco dataset, or give me appropriate loss value?Thanks.

**Comment by zylo117** [comment]
val loss:
cls loss 0.25
reg loss 0.025

**Comment by subhankar-ghosh** [comment]
@zylo117 Can you please provide the exact settings?
Number of epochs? batch size, scheduler, baseLR etc
